apps/client/.env.prod-backend

```
@@ -0,0 +1,12 @@
+# Env preset for running the local dev client against the production
+# backend. Activated by `pnpm dev:prod-backend` (vite --mode prod-backend).
+#
+# URLs point at the Vite dev server's own origin; `server.proxy` in
+# vite.config.ts forwards /__api and /__media to the prod services.
+# This keeps the browser same-origin, sidestepping CORS entirely.
+#
+# Assumes the dev server runs on http://localhost:3510 (PORT=3510 in .env).
+
+PUBLIC_SERVER_URL=http://localhost:3510/__api
+PUBLIC_MEDIASERVER_URL=http://localhost:3510/__media
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_MEDIASERVER_URL key should go before the PUBLIC_SERVER_URL key

(UnorderedKey)